### PR TITLE
HKG: Add FW for 2018 Hyundai Kona EV

### PR DESCRIPTION
**Pull request by jyoung8607**
Add firmware for the 2018 Hyundai Kona EV. Expand supported model-year range.

**Dongle ID:** 3f4095e3f5a6d997

Thanks to community Kona EV owner Jazzah.

### CODE REVIEW

**Comment by jyoung8607** [comment]
The user here is affected by the the same FCA-error-while-engaged malady as the users in #20113 and #20375. At least some of the popular HKG community forks have done something to prevent that from happening, but on cursory inspection I can't see what it is. It doesn't appear to be one of the usual HKG feature-group membership things.

Taking this out of review and proposing a merge because the data is valid, and it's at least as functional as the other listed cars. I'll see if I can track down the issue but it'll be outside of this PR.

**Comment by pd0wm** [comment]
That data in the VIN request is very strange. Especially because that's the first request we send before other UDS interactions. Do you have the rlogs for one of those interactions, would love to see what was going on on the CAN bus.

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ class CAR:
   IONIQ_EV_2020 = "HYUNDAI IONIQ ELECTRIC 2020"
   IONIQ_PHEV = "HYUNDAI IONIQ PHEV 2020"
   KONA = "HYUNDAI KONA 2020"
-  KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
+  KONA_EV = "HYUNDAI KONA ELECTRIC 2018-2019"
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:35]:
```suggestion
  KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
```
We try to not change car strings after the initial port.

**Comment by jyoung8607** [comment]
`3f4095e3f5a6d997|2021-08-27--23-54-57` (preserved, with rlogs)
`3f4095e3f5a6d997|2021-08-28--12-06-31` (preserved, with partial rlogs)

That's pretty much what I have to choose from. The associated C3 is off in retropilot-associated forkworld at this particular moment, but I think he would probably come back if there was something that needed testing or validation.

**Comment by pd0wm** [comment]
Checked the can logs, and it's actually sending that as a response. Probably related to the other weird responses where the ECU isn't fully initialized yet.

```
129 0x7df b'0209020000000000' b'\x02\t\x02\x00\x00\x00\x00\x00'
129 0x18db33f1 b'0209020000000000' b'\x02\t\x02\x00\x00\x00\x00\x00'
0 0x7df b'0209020000000000' b'\x02\t\x02\x00\x00\x00\x00\x00'
2 0x7df b'0209020000000000' b'\x02\t\x02\x00\x00\x00\x00\x00'
1 0x7ec b'10144902014f5345' b'\x10\x14I\x02\x01OSE'
129 0x7e4 b'3000000000000000' b'0\x00\x00\x00\x00\x00\x00\x00'
1 0x7ec b'21565f5f20424d53' b'!V__ BMS'
1 0x7ec b'2220000000000000' b'" \x00\x00\x00\x00\x00\x00'
```

**Pull request merged**